afni_proc_hw/exp_1/proc.sub_03.test

```diff
@@ -0,0 +1,481 @@
+#!/bin/tcsh -xef
+
+echo "auto-generated by afni_proc.py, Mon Aug 14 15:18:06 2017"
+echo "(version 5.15, April 25, 2017)"
+echo "execution started: `date`"
+
+# execute via : 
+#   tcsh -xef proc.sub_03.test |& tee output.proc.sub_03.test
+
+# =========================== auto block: setup ============================
+# script setup
+
+# take note of the AFNI version
+afni -ver
+
+# check that the current AFNI version is recent enough
+afni_history -check_date 23 Sep 2016
+if ( $status ) then
+    echo "** this script requires newer AFNI binaries (than 23 Sep 2016)"
+    echo "   (consider: @update.afni.binaries -defaults)"
+    exit
+endif
+
+# the user may specify a single subject to run with
+if ( $#argv > 0 ) then
+    set subj = $argv[1]
+else
+    set subj = sub_03
+endif
+
+# assign output directory name
+set output_dir = $subj.test-results
+
+# verify that the results directory does not yet exist
+if ( -d $output_dir ) then
+    echo output dir "$subj.test-results" already exists
+    exit
+endif
+
+# set list of runs
+set runs = (`count -digits 2 1 1`)
+
+# create results and stimuli directories
+mkdir $output_dir
+mkdir $output_dir/stimuli
+
+# copy stim files into stimulus directory
+cp /home/charles/Desktop/sub-03/ses-test/correct_task.1D         \
+    /home/charles/Desktop/sub-03/ses-test/incorrect_task.1D      \
+    /home/charles/Desktop/sub-03/ses-test/no_response_control.1D \
+    /home/charles/Desktop/sub-03/ses-test/no_response_task.1D    \
+    /home/charles/Desktop/sub-03/ses-test/response_control.1D    \
+    $output_dir/stimuli
+
+# copy anatomy to results dir
+3dcopy sub-03/ses-test/anat/sub-03_ses-test_T1w.nii.gz \
+    $output_dir/sub-03_ses-test_T1w
+
+# ============================ auto block: tcat ============================
+# apply 3dTcat to copy input dsets to results dir, while
+# removing the first 0 TRs
+3dTcat -prefix $output_dir/pb00.$subj.r01.tcat \
```

Comment:
The first volume is pre steady state, so it should be dropped here. In the proc script, change this to 
`
3dTcat -prefix $output_dir/pb00.$subj.r01.tcat \
sub-03/ses-test/func/sub-03_ses-test_task-linebisection_bold.nii.gz'[1..$]'
`

or specify in `uber_subject` which will add the option `-tcat_remove_first_trs 1` to the `afni_proc.py` command.